android/src/main/java/io/sentry/react/RNSentryModule.java

```diff
@@ -368,17 +368,15 @@ public void addBreadcrumb(final ReadableMap breadcrumb) {
                     case "warning":
                         breadcrumbInstance.setLevel(SentryLevel.WARNING);
                         break;
-                    case "info":
-                        breadcrumbInstance.setLevel(SentryLevel.INFO);
-                        break;
                     case "debug":
                         breadcrumbInstance.setLevel(SentryLevel.DEBUG);
                         break;
                     case "error":
                         breadcrumbInstance.setLevel(SentryLevel.ERROR);
                         break;
+                    case "info":
                     default:
-                        breadcrumbInstance.setLevel(SentryLevel.ERROR);
+                        breadcrumbInstance.setLevel(SentryLevel.INFO);
```

Comment:
Should this be in the changelog as a fix?

Reply:
Nope because it's likely never being set to something else other than the already defined enum values.
It's just feature-proof.